clustering/singleton/extension/src/main/java/org/wildfly/extension/clustering/singleton/SingletonResourceDefinition.java

```diff
@@ -76,8 +80,8 @@
         ;
         private final SimpleAttributeDefinition definition;
 
-        private Attribute(String name, ModelType type, CapabilityReferenceRecorder reference) {
-            this.definition = new SimpleAttributeDefinitionBuilder(name, type)
+        Attribute(String name, ModelType type, CapabilityReferenceRecorder reference) {
+            this.definition = new SimpleAttributeDefinitionBuilder(name, type, true)
```

Comment:
You are right, I am removing this change now. The Jira to support testing this in the test fwk is https://issues.jboss.org/browse/WFCORE-1892 any input there would be appreciated.